extensions/vscode/src/granite/utils/compatibilityUtils.ts

```diff
@@ -53,3 +53,20 @@ export function setupExtensionCheck(
   });
   context.subscriptions.push(disposable);
 }
+
+// Replace Copilot with Granite.Code
+export function replaceCopilotWithGraniteCode() {
+  // Move Copilot to the spare activity sidebar container we created
+  vscode.commands
+    .executeCommand("vscode.moveViews", {
+      viewIds: ["workbench.panel.chat.view.copilot"],
+      destinationId: "workbench.view.extension.copilot",
```

Comment:
Where is this viewcontainer coming from? Does it need to be added in package.json? If that's the correct understanding, I'd prefer an ID more like:

 granite.moveChatDestination

users will of course not see this, but if we're going to be using clever hacks, we have to own up to them.